DigitalLearningSolutions.Web/Extensions/TempDataExtension.cs

```diff
@@ -22,5 +26,18 @@ private static bool TryPeek(this ITempDataDictionary tempData, string key, out s
                     return false;
             }
         }
+
+        private static bool TryGet(this ITempDataDictionary tempData, string key, out string value)

```

Comment:
Would it make sense to return a deserialised object or a null in this method rather than return a bool and assign an empty string to value that's never used?

Reply:
I've just followed the same pattern as the existing Peek/TryPeek method. I'm not really sure why they were written that way in the first place. Do you think I should change both or neither?

Reply:
Nice change!